libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -25,34 +25,55 @@ function(_get_common_test_compile_options output_var c_test flags)
       list(APPEND compile_options "-ffixed-point")
     endif()
 
-    # list(APPEND compile_options "-Wall")
-    # list(APPEND compile_options "-Wextra")
+    list(APPEND compile_options "-Wall")
+    list(APPEND compile_options "-Wextra")
     # -DLIBC_WNO_ERROR=ON if you can't build cleanly with -Werror.
     if(NOT LIBC_WNO_ERROR)
-      # list(APPEND compile_options "-Werror")
+      list(APPEND compile_options "-Werror")
+
+      # Needed because all/most arguments become "unused" when compiling
+      # "libc/test/src/__support/fake_heap.s"
```

Comment:
There's a bug on file for this. Let's refer to that instead.

https://github.com/llvm/llvm-project/issues/122367#issuecomment-2581374103

Reply:
Fixed in: 67ed339cb821